protocol/x/clob/keeper/keeper.go

```diff
@@ -262,6 +263,33 @@ func (k Keeper) InitializeNewStreams(ctx sdk.Context) {
 				clobPairId,
 			)
 		},
+		func(subaccountId satypes.SubaccountId) *satypes.StreamSubaccountUpdate {
+			subaccount := k.subaccountsKeeper.GetSubaccount(
```

Comment:
I think we might need to use `GetSettledSubaccount` to account for funding rates. cc @jayy04

Reply:
This is the subaccount snapshot code. We are accounting for funding rates during the updates.